linker-diff/src/lib.rs

```diff
@@ -172,7 +172,11 @@ impl Config {
                 "section.note.package",
                 // TLSDESC relaxations aren't yet implemented.
                 "rel.match_failed.R_X86_64_GOTPC32_TLSDESC",
-                "rel.R_X86_64_TLSDESC_CALL.R_X86_64_NONE",
+                "rel.missing-opt.R_X86_64_TLSDESC_CALL.SkipTlsDescCall.*",
+                // Wild eliminates GOTPCRELX in statically linked executables even for undefined
+                // symbols, whereas other linkers don't. This is a valid optimisation that other
+                // linkers don't currently do.
+                "rel.extra-opt.R_X86_64_GOTPCRELX.CallIndirectToRelative.static-*",
```

Comment:
Maybe it'd be worth extracting these and AArch64 specific exclusions to the respective modules. On the other hand, having them all in one place makes it easier to find what is excluded.

Reply:
That sounds like something that could be good to do at some stage if the number of architectures grows more and / or the number of architecture-specific exclusions gets larger. Thanks for the suggestion!